runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,57 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({filter})                                       *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range.
+
+    Example usage: >lua
+        local timeout, bufnr = 100, vim.api.nvim_get_current_buf()
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
```

Comment:
To better illustrate the use of the parameter options, I would use `bufnr = 0` here and comment on how that results in the current buffer being used.